cubrid/CMakeLists.txt

```diff
@@ -374,6 +375,8 @@ set(TRANSACTION_HEADERS
   ${TRANSACTION_DIR}/log_recovery_redo.hpp
   ${TRANSACTION_DIR}/log_recovery_redo_parallel.hpp
   ${TRANSACTION_DIR}/log_replication.hpp
+  ${TRANSACTION_DIR}/log_replication_jobs.hpp
+  ${TRANSACTION_DIR}/log_replication.cpp.hpp
```

Comment:
this file has both `.cpp` and `.hpp`

Reply:
that's on purpose; it contains template implementations